### ISSUE TITLE
TestCase Failure : Code formating issue

### ISSUE THREAD

**Issue by Eclesia** [cincheo/jsweet#299]
This test is a little special, since the problem is related to code formating.
The first code fails to transpile, but the second works, the only difference is code formating.

Code failing
```
public class ManyArray {
 
    public static void hash(byte[] m, byte[] h) {
        long[] t = new long[] {0,1};
        for (; t[0] < m.length; t[1] =1)
            block();
        System.arraycopy(m, (int) t[0], new byte[64], 0, m.length - (int)t[0]);
    }
    
    private static void block() {
    }
    
}
```
Code working
```
public class ManyArray {

    public static void hash(byte[] m, byte[] h) {
        long[] t = new long[]{0, 1};
        for (; t[0] < m.length; t[1] = 1) {
            block();
        }
        System.arraycopy(m, (int) t[0], new byte[64], 0, m.length - (int) t[0]);
    }

    private static void block() {
    }

}
```

Error
```
2017-06-09 21:05:53.053 INFO  JSweetTranspiler:1342 - debug/jsweet/jsweettestcase/ManyArray.ts(10,46): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'void' has no compatible call signatures.
2017-06-09 21:05:53.053 ERROR output:48 - cannot invoke an expression whose type lacks a call signature. Type 'void' has no compatible call signatures at /home/husky/NetBeansProjects/JSweetTestCase/src/main/java/debug/jsweet/jsweettestcase/ManyArray.java(17)
```

**Comment by lgrignon** [comment]
Nice one, I reproduced it, I will try to fix it asap. Thanks for reporting.

**Comment by lgrignon** [comment]
Should be fixed via 90e9e00239c6c4daeeada0b3d77c59445fc053fc
Could you please check and close if it is good for you?

**Comment by Eclesia** [comment]
it works now, thanks.
